_data/releases/2021-06/android.yml

```
@@ -0,0 +1,103 @@
+entries:
+- Name: azure-core
+  Version: 1.0.0-beta.6
+  DisplayName: Core
+  ServiceName: Core
+  VersionType: Beta
+  Hidden: false
+  ChangelogUrl: https://github.com/Azure/azure-sdk-for-android/tree/azure-core_1.0.0-beta.6/sdk/core/azure-core/CHANGELOG.md
+  ChangelogContent: |-
+    #### New Features
+
+    - Added `AsyncStream` and `AsyncStreamHandler` to support enumeration of a stream of data asynchronously.
+    - Added `Page` and `PagedAsyncStreamCore` that defines the paging contract.
+
+    #### Breaking Changes
+
+    - Renamed `azure.core.util.Context` to `azure.core.util.RequestContext`.
+    - Removed `ContinuablePage`.
+  GroupId: com.azure.android
+- Name: azure-core-http-okhttp
+  Version: 1.0.0-beta.6
+  DisplayName: Core HTTP OkHttP
```

I probably need to correct this in our packages .csv file
```suggestion
  DisplayName: Core HTTP OkHttp
```